src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -82,5 +82,8 @@
             <argument /> <!-- multiplier -->
             <argument /> <!-- max delay ms -->
         </service>
+        <service id="messenger.transport.in_memory.factory" class="Symfony\Component\Messenger\Transport\InMemoryTransportFactory">
```

Comment:
Can you add a space between the `service`s and move it next to the other transports?

```suggestion

        <service id="messenger.transport.in_memory.factory" class="Symfony\Component\Messenger\Transport\InMemoryTransportFactory">
```